app/locales/en/translations.js

```diff
@@ -78,5 +93,47 @@ export default {
       saved_title: 'Imaging Request Saved',
       saved_message: 'The imaging request has been saved.'
     }
+  },
+  medication: {
+    page_title: 'Medication Requests',
+    section_title: 'Medication',
+    return_medication: 'Return Medication',
+    buttons: {
+      dispense_medication: 'dispense medication',
+      new_button: '+ new request',
+      return_medication: 'return medication'
+    },
+    titles: {
+      completed_medication: 'Completed Medication',
+      edit_medication_request: 'Edit Medication Request',
+      new_medication_request: 'New Medication Request'
+    },
+    messages: {
+      create_new: 'Create a new medication request?',
+      confirm_deletion: 'Are you sure you wish to delete this medication request?'
+    },
+    labels: {
+      refills: 'Refills',
+      quantity_requested: 'Quantity Requested',
+      quantity_dispensed: 'Quantity Dispensed',
+      quantity_distributed: 'Quantity Distributed',
+      quantity_to_return: 'Quantity To Return',
+      return_location: 'Return Location',
+      return_aisle: 'Return Aisle',
+      return_reason: 'Return Reason/Notes',
+      adjustment_date: 'Adjustment Date',
+      credit_to_account: 'Credit To Account'
+    },
+    alerts: {
+      returned_title: 'Medication Returned',
+      returned_message:  'The medication has been marked as returned.',
+      saved_title: 'Medication Request Saved',
+      saved_message: 'The medication record has been saved.',
+      fulfilled_title: 'Medication Request Fulfilled'
+    },
+    common: {
```

Comment:
Also, if it is intended to be app wide, a label other than "common" would be better.